### ISSUE TITLE
SR, growth factor and rate

### ISSUE THREAD

**Issue by marcobonici** [CosmologicalEmulators/Effort.jl#35]
In this moment, we are solving equations to get $D(z)$ and $f(z)$ as a function of cosmological parameters.
Another option is to use Symbolic Regression.
Regarding $D(z)$, its usage would be clear: we are using it to make a smart rescaling, hence even if it is not super accurate (5%?) it would still be good as we would still get most of the benefits, but we would gain in performance (symbolic expressions are fast) and in term of reproducibility (this avoids having problems like a change in `SciML` stuff).
Regarding $f(z)$, it is a bit more subtle. Some EFT codes, such as `pybird`, requires `f` when doing their bias contraction. Up to now, this was our weakest spot: `pybird` was using `class` for that, and we had some small discrepancies (nothing worrisome, but it is annoying to have an emulator with 0.1% accuracy and then `f` is worse...). The strategy would be to take the component of the spectra that gets multiplied by `f` and multiply it by `f_class` / `f_SR`. Then, at runtime, we would multiply by `f_SR`. In this way, we will get speed and accuracy.

Finally, this will help with the `jax` version. Although we can use the python version of `OrdinaryDifferentialEquations.jl`, we won't be able to differentiate through it. Although we might use `diffrax`, which would ensure AD, performance etc, it would likely give _slightly_ different results from the `julia` version. In this way, we will ensure speed, precision, reproducibility and complete equivalence among `jaxeffort` and `Effort.jl`.

**Comment by marcobonici** [comment]
In the end, we got the ODE working and precise in jax. We are closing this issue.
